models/engine/db_storage.py

```diff
@@ -74,3 +74,15 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """ get an item with a specific class and id """
+        return next((
+            value for key, value
+            in self.all(cls).items()
+            if key.split(".")[-1] == id
+        ), None)
```

Comment:
There is no reason for this at all. It's hard to read and needlessly complex. The majority of coders have no idea what next() does, or the inner workings of it. This function is a five line breeding ground for bugs.